handler/handler.go

```diff
@@ -1,15 +1,24 @@
 package handler
 
 import (
+	"github.com/gin-gonic/gin"
 	"github.com/murilogilfelpeto/ps-tag-onboarding-go/configuration"
 	"github.com/murilogilfelpeto/ps-tag-onboarding-go/service"
 )
 
-var (
-	logger *configuration.Logger
-)
+var logger = configuration.NewLogger("handler")
+
+type Handler interface {
+	Save(context *gin.Context)
+	FindById(context *gin.Context)
+}
+
+type handler struct {
+	service service.Service
+}
```

Comment:
Is this the user handler? You could keep this struct near where it's receivers are defined.

Reply:
this the handler, the userHandler is another file.